group-coordinator/src/test/java/org/apache/kafka/coordinator/group/AssignmentTestUtil.java

```diff
@@ -85,19 +86,20 @@ public static void assertAssignment(
     }
 
     /**
-     * Generate a reverse look up map of partition to member target assignments from the given member spec.
+     * Generate a reverse look up map of partition to member target assignments from the given metadata.
      *
-     * @param memberSpec        A map where the key is the member Id and the value is an
-     *                          AssignmentMemberSpec object containing the member's partition assignments.
+     * @param assignedPartitions              Partition assignments per member.
+     * @param memberSubscriptionSpec          A map where the key is the member Id and the value is a
+     *                                        MemberSubscriptionSpec object containing the member's subscription info.
      * @return Map of topic partition to member assignments.
      */
     public static Map<Uuid, Map<Integer, String>> invertedTargetAssignment(
-        Map<String, AssignmentMemberSpec> memberSpec
+        Map<String, Map<Uuid, Set<Integer>>> assignedPartitions,
+        Map<String, MemberSubscriptionSpec> memberSubscriptionSpec
```

Comment:
Do we really need `memberSubscriptionSpec`? It seems that we only use it to get the member ids but we also have them in `assignedPartitions`, no?